tests/python/test_bit_struct.py

```diff
@@ -0,0 +1,68 @@
+import taichi as ti
+import numpy as np
+
+
+def test_simple_array():
+    ti.init(arch=ti.cpu, debug=True, print_ir=True, cfg_optimization=False)
+    ci13 = ti.type_factory_.get_custom_int_type(13, True)
+    cu19 = ti.type_factory_.get_custom_int_type(19, False)
+
+    x = ti.field(dtype=ci13)
+    y = ti.field(dtype=cu19)
+
+    N = 12
+
+    ti.root.dense(ti.i, N)._bit_struct(num_bits=32).place(x, y)
+
+    ti.get_runtime().materialize()
+
+    @ti.kernel
+    def set_val():
+        for i in range(N):
+            x[i] = -2**i
+            y[i] = 2**i - 1
+
+    @ti.kernel
+    def verify_val():
+        for i in range(N):
+            assert x[i] == -2**i
+            assert y[i] == 2**i - 1
+
+    set_val()
+    verify_val()
+
+
+def test_custom_int_load_and_store():
+    ti.init(arch=ti.cpu, debug=True, print_ir=True, cfg_optimization=False)
+    ci13 = ti.type_factory_.get_custom_int_type(13, True)
+    cu14 = ti.type_factory_.get_custom_int_type(14, False)
+    ci5 = ti.type_factory_.get_custom_int_type(5, True)
+
+    x = ti.field(dtype=ci13)
+    y = ti.field(dtype=cu14)
+    z = ti.field(dtype=ci5)
+
+    test_case_np = np.array([[2**12 - 1, 2**14 - 1, -(2**3)],
+                             [2**11 - 1, 2**13 - 1, -(2**2)], [0, 0, 0]],
+                            dtype=np.int32)
```

Comment:
It's nice to test the boundary values of those CustomIntTypes. How about adding a handy extension in this test case to cover another group of `test_case_np` with non-boundary values?

Reply:
Sure! I will add some other test cases.